src/pages/PostList/PostList.scss

```
@@ -0,0 +1,52 @@
+.postList {
+  margin: 0 auto;
+  display: flex;
+  width: 576px;
+  height: 80vh;
+  padding: 40px 24px;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 16px;
+  border: 1px solid #e5e5e5;
+  background: white;
+
+  .posts {
+    overflow-y: scroll;
+    flex: 1;
+  }
+  .posts::-webkit-scrollbar {
+    display: none;
+  }
```

부모 선택자(`&`)를 사용해서 해당 요소에만 적용되는 스타일임을 좀 더 깔끔하게 명시할 수 있습니다!
```suggestion
  .posts {
    overflow-y: scroll;
    flex: 1;
    
    &::-webkit-scrollbar {
      display: none;
    }
  }
```